test/relations.jl

```diff
@@ -1032,6 +1292,10 @@ end
     for ArrayType in array_types
         FT = eltype(ArrayType)
         param_set = TP.ThermodynamicsParameters(FT)
+        _cp_d = TP.cp_d(param_set)
+        _LH_v0 = TP.LH_v0(param_set)
+        _eint_v0 = TP.e_int_v0(param_set)
+        _T_0 = TP.T_0(param_set)
```

Comment:
Do we need anything other than _eint_v0?

Reply:
No, good point. Removed the others.